romancal/lib/basic_utils.py

```diff
@@ -1,43 +1,77 @@
 """General utility objects"""
-import re
 
 
-def multiple_replace(string, rep_dict):
-    """Single-pass replacement of multiple substrings
+def bytes2human(n):
```

Comment:
This is used in the JWST logging messages to make the output more readable, and is specifically used for some of the memory error issues. Since I wrote a Jira ticket for us to start looking at memory issues I thought I just go ahead and include this function. I can remove it and add it back later. This is to stop us from writing an independent function that does the same thing.